.github/workflows/continuous-integration-smoketest.yml

```diff
@@ -34,12 +45,17 @@ jobs:
             -DKokkos_ENABLE_EXAMPLES=ON \
             -DKokkos_ENABLE_SERIAL=ON \
             -DKokkos_ENABLE_TESTS=ON \
+            -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
+            -DCMAKE_BUILD_TYPE=Release \
```

Comment:
Update L39 instead

Reply:
Dah! Will do!  Let first see, how much is actually helps....

Reply:
I would slightly prefer `RelWithDebInfo` btw ;)

Reply:
Done ;-)